### PR TITLE
made list of liquids and gases global in fluids.py

### PR DESCRIPTION
**Pull request by dlohmeier**
A first step towards an applicable fluid library, if needed. Anyone from outside at least can get a better overview of the implemented fluids in pandapipes.

### CODE REVIEW

**Comment by dlohmeier** [comment]
@kbensafta could you provide a lhv for the new biomethane properties? You can just write them here, I could add them.

**Review by SimonRubenDrauz — APPROVED** [review]

File: pandapipes/properties/fluids.py [hunk 1]
```diff
@@ -668,36 +671,31 @@ def linear_property(prop):
...
+    properties["molar_mass"] = constant_property("molar_mass")
+    properties["der_compressibility"] = constant_property("der_compressibility")
+    properties["compressibility"] = linear_property("compressibility")
+
+    if phase == 'gas':
+        for entry, name in [("lhv", "lower_heating_value"), ("hhv", "higher_heating_value")]:
+            try:
+                properties[entry] = constant_property(name)
+            except FileNotFoundError:
+                pass
```
Comment (SimonRubenDrauz) [inline pandapipes/properties/fluids.py:696]: I would add a Warning here.

**Review by SimonRubenDrauz — DISMISSED** [review]
I really like the implementation. I just would add a Warning in case of not finding lhv/hhv!

File: pandapipes/properties/fluids.py (hunk 1 above)
Comment (dlohmeier) [reply pandapipes/properties/fluids.py]: warning was added.

**Pull request merged**